pokus.txt

```diff
@@ -0,0 +1 @@
+rwer
```

Comment:
ted bych ti to tady nejak pohejtil. - vidis to? kdyztak mi na to i odpovez

Reply:
A já tě odhejtuju zpátky

Reply:
jo presne tak to funguje.. tady probiha kolem toho kodu nejak konverzce.. a az sou oba spokojeni tak se to vyrezolvuje ten thread